runtime/vm/jvminit.c

```diff
@@ -3384,110 +3384,116 @@ zeroInitStages(J9JavaVM* vm, IDATA stage, void* reserved)
 {
 	J9VMDllLoadInfo* loadInfo;
 	IDATA returnVal = J9VMDLLMAIN_OK;
-	IDATA argIndex1;
+	IDATA argIndex1 = -1;
 	BOOLEAN describe = FALSE;
 
 	PORT_ACCESS_FROM_JAVAVM(vm);
 
 	switch(stage) {
 		case PORT_LIBRARY_GUARANTEED :
-			vm->zeroOptions = J9VM_ZERO_SHAREBOOTZIPCACHE;
-			argIndex1 = FIND_ARG_IN_VMARGS_FORWARD(STARTSWITH_MATCH, VMOPT_XZERO, NULL);
-			while(argIndex1 >= 0) {
-				char *optionString;
-				char optionsBuffer[LARGE_STRING_BUF_SIZE];
-				char* optionsBufferPtr = (char*)optionsBuffer;
-				UDATA rc;
-
-				optionString = vm->vmArgsArray->actualVMArgs->options[argIndex1].optionString;
-				if (strcmp(optionString, VMOPT_XZERO) == 0) {
-					/* Found -Xzero, enable the default Zero options */
+			/* -Xzero option is removed from Java 9 */
+			if (J2SE_VERSION(vm) >= J2SE_19) {
+				vm->zeroOptions = 0;
+			}
+			else {
```

Comment:
Please put "}" and "else {" in the same line

Reply:
fixed